### ISSUE TITLE
Communicating "imageLibrary" in metadata for segmentation converters

### ISSUE THREAD

**Issue by che85** [QIICR/dcmqi#105]
the same way it is done in https://github.com/QIICR/dcmqi/blob/master/doc/sr-tid1500-ct-liver-example.json

**Comment by fedorov** [comment]
so in SEG it is a little bit different, and that is why this issue may need more discussion.

In SR, "Image library" has a specific meaning - it is a section of the report that summarizes important characteristics of the images on which reporting is made, with the purpose of being able to look up those characteristics without having access to the referenced images.

In SEG, we need source image instances to include references to UIDs, and to inherit the metadata related to study/patient. I don't have a good idea how to reconcile these two concepts, and it would be good to brainstorm this issue with @dclunie, probably at RSNA, or after RSNA. I think RSNA is a great opportunity for the discussion of how to move forward with JSON schemas, since @hmeine and @nolden will be there. This topic is also somewhat related to communicating metadata without having to pass source DICOMs to the converter at all considered in #54.

In short, I would suggest to keep this for RSNA discussion.

**Comment by che85** [comment]
Alright. I will create a label for "After RSNA"

**Comment by che85** [comment]
Closing this issue since we are providing the option to communicate a whole directory now next to a list of DICOM files.
